ferrocene/doc/internal-procedures/src/testing-other-targets.rst

```diff
@@ -138,6 +138,11 @@ Target Procedures
 
 Currently bare metal targets have a similar procedure for testing.
 
+.. note::
+
+   Currently, these targets use our *secret sauce*.
```

Comment:
Does **these** refer to bare metal? Or to `aarch64-unknown-none`?

I saw some Zulip chat a while back about how we want to talk about 'secret sauce', but can't recall the details.

Reply:
`aarch64-unknown-none` is a bare metal target

regarding the_sauce_, this pr only moves the text around (and I don't remember what those details could be)